test/Kongverge.Tests/Services/ConfigFileReaderTests.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Kongverge.DTOs;
+using Kongverge.Services;
+using Nito.AsyncEx;
+using TestStack.BDDfy;
+using TestStack.BDDfy.Xunit;
+
+namespace Kongverge.Tests.Services
+{
+    [Story(Title = nameof(ConfigFileReader))]
+    public class ConfigFileReaderTests : ScenarioFor<ConfigFileReader>
+    {
+        private const string ExceptionThrownWithErrorMessages = "an exception is thrown showing error messages attributed to their respective file paths";
+        private const string TestPath = nameof(TestPath);
+
+        public ConfigFileReaderTests() => Use<IFileProvider>(new VirtualFileProvider());
+
+        [BddfyFact(DisplayName = nameof(ConfigFileReader.ReadConfiguration) + "MoreThanOneGlobalFile")]
+        public void Scenario1()
```

Comment:
Can we use descriptive names for the tests.

Reply:
See answer above.